src/sites/sync.rs

```diff
@@ -59,82 +57,5 @@ pub fn sync(
         .collect();
 
     StdErr::success("Success");
-    Ok((to_upload, to_delete, asset_manifest))
-}
-
-fn filter_files(pairs: Vec<KeyValuePair>, already_uploaded: &HashSet<String>) -> Vec<KeyValuePair> {
-    let mut filtered_pairs: Vec<KeyValuePair> = Vec::new();
-    for pair in pairs {
-        if !already_uploaded.contains(&pair.key) {
-            filtered_pairs.push(pair);
-        }
-    }
-    filtered_pairs
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use crate::sites::generate_path_and_key;
-    use std::collections::HashSet;
-    use std::path::Path;
-
-    #[test]
-    fn it_can_filter_preexisting_files() {
```

Comment:
I guess this would be too painful to make work with `directory_keys_values` for now? Would be good to add back in later with a mock Walk or something.

Reply:
Yea, exactly -- the old test really didn't prove much... and with the logic embedded in the other function now it's a bit hairy to test. We'd want to change the implementation to make it testable but since this is just a quick patch I think it's ok. 

cc/ @jyn514  (w/r/t your question above)